sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -167,6 +167,71 @@ case class MapKeys(child: Expression)
     copy(child = newChild)
 }
 
+
+/**
+ * Returns an unordered array containing the keys of the map.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(map) - Returns true if the map contains the key.",
```

Comment:
The function has 2 arguments. Please, add `key`.

Reply:
updated